packages/bpk-component-map/index.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import BpkMap from './src/BpkMap';
+import BpkMapMarker from './src/BpkMapMarker';
+
+export default BpkMap;
+export { BpkMapMarker };
+export * from 'react-google-maps';
```

Comment:
How come we need to export all of the exports in `react-google-maps`?

Reply:
Well, I felt that it would be beneficial to give consumers access to everything `react-google-maps` offers. There's quite a lot of exports (it's pretty powerful). 

I'm thinking of a situation where say, a consumer wants to draw a circle on the map (let's say it's to draw a radius around a hotel). If we're not exporting that, they'd have to pull it in from `react-google-maps` instead which doesn't feel quite right to me.

I am totally open to opinions here - I don't think it's something we've done before and we should make sure we get it right. I mentioned it in the 'open questions' part of the PR description:

> Is `export * from 'react-google-maps` ok? I don't know of anywhere else we do this. Alternatively we could export everything from the package manually.

Reply:
Sounds good to me 👍